src/leap/mail/imap/account.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, user_id, store, d=defer.Deferred()):
         # about user_id, only the client backend.
 
         self.user_id = user_id
-        self.account = Account(store, ready_cb=lambda: d.callback(self))
+        self.account = Account(store, user_id, ready_cb=lambda: d.callback(self))
```

Comment:
nevermind, maybe it's easier to change the IMAPAccount order. I'll merge, we can discuss further changes later.